### ISSUE TITLE
PaginationNav is not accessible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6891]
## Environment

> Operating system

Windows 10 latest

> Browser

Edge, Firefox

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description
The right and left arrow on firefox is too low contrast, but strangely on Edge only the right arrow is affected. On firefox there are no focus states

Firefox
![image](https://user-images.githubusercontent.com/40970507/93943564-6aac7f00-fcf8-11ea-93b8-0bf82d3147a8.png)

Edge
![image](https://user-images.githubusercontent.com/40970507/93943576-6f713300-fcf8-11ea-92a3-ab8f9f169891.png)

**Comment by rjhenriquez** [comment]
@dakahn This was fixed [here](https://github.com/carbon-design-system/carbon/pull/6892)
I'm closing the issue

![Screen Shot 2020-10-13 at 10 16 11 AM](https://user-images.githubusercontent.com/4718579/95880518-35b2ab80-0d3d-11eb-98db-a4dce64b1778.png)
